app/scripts/modules/core/delivery/triggers/TriggersTag.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { filter } from 'lodash';
+
+import { IPipeline } from 'core/domain/IPipeline';
+
+interface IProps {
+  pipeline: IPipeline;
+};
+
+interface IState {
+  triggerCount: number;
+  activeTriggerCount: number;
+};
+
+export class TriggersTag extends React.Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+
+    let triggerCount = 0,
+        activeTriggerCount = 0;
+
+    const pipeline = this.props.pipeline;
+    if (pipeline && pipeline.triggers && pipeline.triggers.length) {
+      triggerCount = pipeline.triggers.length;
+      activeTriggerCount = filter(pipeline.triggers, { enabled: true }).length;
+    }
+
+    this.state = {
+      triggerCount,
+      activeTriggerCount,
+    }
+  }
+
+  public render(): React.ReactElement<TriggersTag> {
+    const triggerCount = this.state.triggerCount,
+          activeTriggerCount = this.state.activeTriggerCount;
+
+    if (triggerCount > 0) {
+      const triggers = triggerCount === 1 ? 'Trigger' : activeTriggerCount === triggerCount ? 'All triggers' : 'Some triggers';
+      const displayTriggers = `${triggers}: ${activeTriggerCount === 0 ? 'disabled' : 'enabled'}`;
+      return (
+        <div className={`triggers-toggle ${activeTriggerCount ? '' : 'disabled'}`}>
+          <span>
+            <span>
+              { displayTriggers }
+            </span>
+          </span>
+        </div>
+      );
+    }
+    return <div></div>;
```

Comment:
Since the valid case returns a `<div>`, I don't think there's concern. I'd rather render it the same style and fix any resulting bugs from that then try to render something different... It all renders appropriately, afaict.